networks

```diff
@@ -5,27 +5,21 @@ avalanche
 base
 boba
 bsc
-celo
-cronos
-evmos
 fantom
 fuse
 gnosis
 linea
-localanvil
+metis
 moonbeam
 moonriver
-okx
-opbnb
 optimism
 polygon
 polygonzkevm
-velas
+scroll
+zksync
 goerli
```

Comment:
why are we keeping goerli? This network is not functional anymore. Would recommend to remove it